com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfJoints/UrdfJointPlanar.cs

```diff
@@ -109,61 +71,32 @@ private static SoftJointLimit GetLinearLimit(Joint.Limit limit)
 
         protected override Joint ExportSpecificJointData(Joint joint)
         {
-#if UNITY_2020_1_OR_NEWER
             joint.axis = GetAxisData(axisofMotion);
             joint.dynamics = new Joint.Dynamics(unityJoint.linearDamping, unityJoint.jointFriction);
             joint.limit = ExportLimitData();
-#else
-            ConfigurableJoint configurableJoint = (ConfigurableJoint)unityJoint;
-            joint.axis = GetAxisData(Vector3.Cross(configurableJoint.axis, configurableJoint.secondaryAxis));
-            joint.dynamics = new Joint.Dynamics(configurableJoint.xDrive.positionDamper, configurableJoint.xDrive.positionSpring);
-            joint.limit = ExportLimitData();
-#endif
             return joint;
         }
 
         protected override Joint.Limit ExportLimitData()
         {
-#if UNITY_2020_1_OR_NEWER
             ArticulationDrive drive = GetComponent<ArticulationBody>().yDrive;
             return new Joint.Limit(drive.lowerLimit, drive.upperLimit, EffortLimit, VelocityLimit);
-#else
-            ConfigurableJoint configurableJoint = (ConfigurableJoint)unityJoint;
-            return new Joint.Limit(
-                Math.Round(-configurableJoint.linearLimit.limit, RoundDigits),
-                Math.Round(configurableJoint.linearLimit.limit, RoundDigits),
-                EffortLimit, VelocityLimit);
-#endif
         }
 
         public override bool AreLimitsCorrect()
         {
-#if UNITY_2020_1_OR_NEWER
             ArticulationBody joint = GetComponent<ArticulationBody>();
             return joint.linearLockY == ArticulationDofLock.LimitedMotion &&
                 joint.linearLockZ == ArticulationDofLock.LimitedMotion &&
                 joint.yDrive.lowerLimit < joint.yDrive.upperLimit &&
                 joint.zDrive.lowerLimit < joint.zDrive.upperLimit;
-#else
-            ConfigurableJoint joint = (ConfigurableJoint)unityJoint;
-            return joint != null && joint.linearLimit.limit != 0;
-#endif
         }
 
         protected override bool IsJointAxisDefined()
         {
-#if UNITY_2020_1_OR_NEWER
+            // Vidur - what is this??
```

Comment:
@vidurvij-Unity ^^

Reply:
Same reasoning as above. This should be removed.

Reply:
Got it. Will do.